docs/source/integrations/v7.rst

```diff
@@ -0,0 +1,968 @@
+
+    dataset = foz.load_zoo_dataset(
+        "quickstart", dataset_name="v7-annotation-example"
+    )
+    dataset.persistent = True
+
+    dataset.evaluate_detections(
+        "predictions", gt_field="ground_truth", eval_key="eval"
+    )
+
+    # Step 2: Locate a subset of your data requiring annotation
+
+    # Create a view that contains only high confidence false positive model
+    # predictions, with samples containing the most false positives first
+    most_fp_view = (
+        dataset
+        .filter_labels("predictions", (F("confidence") > 0.8) & (F("eval") == "fp"))
+        .sort_by(F("predictions.detections").length(), reverse=True)
+    )
+
+    # Retrieve the sample with the most high confidence false positives
+    sample_id = most_fp_view.first().id
+    view = dataset.select(sample_id)
+
+    # Step 3: Send samples to V7
+
+    # A unique identifier for this run
+    anno_key = "v7_basic_recipe"
+
+    label_schema = {
+        "new_ground_truth": {
+            "type": "detections",
+            "classes": dataset.distinct("ground_truth.detections.label"),
+            "attributes": {
+                "iscrowd": {
+                    "type": "radio",
+                    "values": [True, False],
+                },
+            },
+        },
+    }
+
+    view.annotate(
+        anno_key,
+        backend="darwin",
+        label_schema=label_schema,
+        launch_editor=True,
+    )
+    print(dataset.get_annotation_info(anno_key))
+
+    # Step 4: Perform annotation in V7 and save the tasks
+
+Then, once the annotation work is complete, we merge the annotations back into
+FiftyOne:
+
+.. code-block:: python
+    :linenos:
+
+    import fiftyone as fo
+
+    anno_key = "v7_basic_recipe"
+
+    # Step 5: Merge annotations back into FiftyOne dataset
+
+    dataset = fo.load_dataset("v7-annotation-example")
+    dataset.load_annotations(anno_key)
+
+    # Load the view that was annotated in the App
+    view = dataset.load_annotation_view(anno_key)
+    session = fo.launch_app(view=view)
+
+    # Step 6: Cleanup
+
+    # Delete tasks from V7
+    results = dataset.load_annotation_results(anno_key)
+    results.cleanup()
+
+    # Delete run record (not the labels) from FiftyOne
+    dataset.delete_annotation_run(anno_key)
+
+.. note::
+
+    See :ref:`this section <v7-examples>` to see a variety of common V7
+    annotation patterns.
+
+.. _v7-setup:
+
+Setup
+_____
+
+You can get started with V7 by
+`creating an account <https://www.v7labs.com/sign-up>`_ and downloading an
+API key.
+
+Installing the V7 backend
+-------------------------
+
+In order to use the V7 backend, you must install the ``fiftyone_darwin`` Python
+package:
+
+.. code-block:: shell
+
+    pip install fiftyone_darwin
+
+and register the ``darwin`` backend with FiftyOne, which you can do either by
+setting the following environment variables:
+
+.. code-block:: shell
+
+    export FIFTYONE_ANNOTATION_BACKENDS=*,darwin
```

Comment:
Is this syntax (with the asterisk) supported literally to mean eg keep default backends" ?

Reply:
Yeah I just recently added support for this syntax here: https://github.com/voxel51/fiftyone/commit/6ab2ac970c956a16555ee222a2266bb2dccaa7a7

Unfortunately it won't be available until `fiftyone==0.24` is released though, so you should probably revert to explicitly listing the builtin backends for now